chromium_src/components/search_engines/brave_template_url_service_util_unittest.cc

```diff
@@ -45,15 +46,47 @@ class BraveTemplateURLServiceUtilTest : public testing::Test {
   sync_preferences::TestingPrefServiceSyncable prefs_;
 };
 
-TEST_F(BraveTemplateURLServiceUtilTest, GetSearchProvidersUsingKeywordResult) {
+void TestDefaultOrder(TemplateURL::OwnedTemplateURLVector& template_urls,
+                      std::vector<std::string> keywords) {
+  EXPECT_EQ(template_urls.size(), keywords.size());
+
+  for (int i = 0; i < (int)template_urls.size(); i++) {
+    EXPECT_EQ(template_urls[i]->keyword(), base::ASCIIToUTF16(keywords[i]));
+  }
+}
+
+std::vector<TemplateURLData> GetSampleTemplateData() {
   std::vector<TemplateURLData> local_turls;
+
   // Create a sets of TURLs in order different from prepopulated TURLs.
-  local_turls.push_back(*CreatePrepopulateTemplateURLData(511, ":sp", 1));
+  local_turls.push_back(*CreatePrepopulateTemplateURLData(
+      TemplateURLPrepopulateData::PREPOPULATED_ENGINE_ID_STARTPAGE, ":sp", 1));
   local_turls.push_back(*CreatePrepopulateTemplateURLData(15, ":ya", 2));
-  local_turls.push_back(*CreatePrepopulateTemplateURLData(3, ":b", 3));
-  local_turls.push_back(*CreatePrepopulateTemplateURLData(507, ":q", 4));
-  local_turls.push_back(*CreatePrepopulateTemplateURLData(501, ":d", 5));
-  local_turls.push_back(*CreatePrepopulateTemplateURLData(1, ":g", 6));
+  local_turls.push_back(*CreatePrepopulateTemplateURLData(
+      TemplateURLPrepopulateData::PREPOPULATED_ENGINE_ID_BING, ":b", 3));
+  local_turls.push_back(*CreatePrepopulateTemplateURLData(
+      TemplateURLPrepopulateData::PREPOPULATED_ENGINE_ID_QWANT, ":q", 4));
+  local_turls.push_back(*CreatePrepopulateTemplateURLData(
+      TemplateURLPrepopulateData::PREPOPULATED_ENGINE_ID_DUCKDUCKGO, ":d", 5));
+  local_turls.push_back(*CreatePrepopulateTemplateURLData(
+      TemplateURLPrepopulateData::PREPOPULATED_ENGINE_ID_GOOGLE, ":g", 6));
+
+  return local_turls;
+}
+
+WDKeywordsResult GetKeywordResult(
```

Comment:
I'd name it `InitKeywordResult` or `GetExpectedKeywordResult`, atm the name is a bit misleading

Reply:
Fixed! 😄